### DISCUSSION TITLE
Can I update value outside the component?

### DISCUSSION THREAD

**Discussion by jatinhemnani01** [preactjs/preact#3730] [General]
I wanna update the value of state outside the component, is that possible?

**Comment by rschristian** [comment]
Can you provide a bit more detail/explanation of what you're looking for?

State setters can be passed to other components and there's a whole plethora of ways to manage state outside of components altogether, examples including Redux, Mobx, and our recently released [signals](https://preactjs.com/guide/v10/signals) (amongst many others).

State outside of components can't quite be as simple as just a variable, as we need some way to signal a rerender or other update mechanism.
